ci/win32/profile.cmd

```diff
@@ -0,0 +1,12 @@
+set profile_results_file=profiling_result.prof
+set esdl_file=.\testdata\test1.esdl
+
+pushd .
+cd /D "%~dp0"
+
+cd ..\..\
+call .\venv\Scripts\activate
+set PYTHONPATH=.\src\;%$PYTHONPATH%
+python .\src\simulator_core\infrastructure\profiling.py %esdl_file% %profile_results_file%
```

Comment:
Ik kan niet verifieren of deze code doet wat hij moet doen, omdat mijn simulator-core package kapot is.